arnold_usd.h.in

```diff
@@ -19,4 +19,4 @@
 
 #define USED_USD_VERSION_GREATER_EQ(x, y) (USED_USD_MINOR_VERSION >= x && USED_USD_PATCH_VERSION >= y ) || USED_USD_MINOR_VERSION > x || USED_USD_MAJOR_VERSION > 0
 
-#define AI_VERSION_NUMBER (${ARNOLD_VERSION_ARCH_NUM} * 10000 + ${ARNOLD_VERSION_MAJOR_NUM} * 100 + ${ARNOLD_VERSION_MINOR_NUM})
+#define ARNOLD_VERSION_NUMBER (${ARNOLD_VERSION_ARCH_NUM} * 10000 + ${ARNOLD_VERSION_MAJOR_NUM} * 100 + ${ARNOLD_VERSION_MINOR_NUM})
```

Comment:
There's already a similar define in `ai_version.h`. Sadly it does not follow the naming convention as well, I've made a comment in ARNOLD-10088 to remember to address this.

Reply:
I would prefer to wait until the core define gets renamed, we can either keep this pull request open until then or create a second pull request to use the Arnold define and possibly get rid of arnold_usd.h. (since we won't be using it anymore)